core/src/main/java/io/ap4k/project/Project.java

```diff
@@ -74,18 +74,26 @@ public Project withAp4kOutputDir(String ap4kOutputDir) {
     return new Project(root, ap4kInputDir, ap4kOutputDir, buildInfo);
   }
 
-  public Map<String, Object> parseAppResourceFile(String resourceName) throws IOException {
```

Comment:
Let's just call it `parseResource` as it could actually parse any resource.

Reply:
Fixed